test/ecdsatest.h

```diff
@@ -0,0 +1,10214 @@
+    "9d0c6afb6df3bced455b459cc21387e14929392664bb8741a3693a1795ca6902",
+    "d7f9ddd191f1f412869429209ee3814c75c72fa46a9cccf804a2f5cc0b7e739f"},
+    {NID_X9_62_prime256v1, NID_sha256,
+    "558c2ac13026402bad4a0a83ebc9468e50f7ffab06d6f981e5db1d082098065bcff6f21a"
+    "7a74558b1e8612914b8b5a0aa28ed5b574c36ac4ea5868432a62bb8ef0695d27c1e3ceaf"
+    "75c7b251c65ddb268696f07c16d2767973d85beb443f211e6445e7fe5d46f0dce70d58a4"
+    "cd9fe70688c035688ea8c6baec65a5fc7e2c93e8",
+    "5e666c0db0214c3b627a8e48541cc84a8b6fd15f300da4dff5d18aec6c55b881",
+    "041bc487570f040dc94196c9befe8ab2b6de77208b1f38bdaae28f9645c4d2bc3aec8160"
+    "2abd8345e71867c8210313737865b8aa186851e1b48eaca140320f5d8f",
+    "2e7625a48874d86c9e467f890aaa7cd6ebdf71c0102bfdcfa24565d6af3fdce9",
+    "2f9e2b4e9f747c657f705bffd124ee178bbc5391c86d056717b140c153570fd9",
+    "f5413bfd85949da8d83de83ab0d19b2986613e224d1901d76919de23ccd03199"},
+    {NID_X9_62_prime256v1, NID_sha256,
+    "4d55c99ef6bd54621662c3d110c3cb627c03d6311393b264ab97b90a4b15214a5593ba25"
+    "10a53d63fb34be251facb697c973e11b665cb7920f1684b0031b4dd370cb927ca7168b0b"
+    "f8ad285e05e9e31e34bc24024739fdc10b78586f29eff94412034e3b606ed850ec2c1900"
+    "e8e68151fc4aee5adebb066eb6da4eaa5681378e",
+    "f73f455271c877c4d5334627e37c278f68d143014b0a05aa62f308b2101c5308",
+    "04b8188bd68701fc396dab53125d4d28ea33a91daf6d21485f4770f6ea8c565dde423f05"
+    "8810f277f8fe076f6db56e9285a1bf2c2a1dae145095edd9c04970bc4a",
+    "62f8665fd6e26b3fa069e85281777a9b1f0dfd2c0b9f54a086d0c109ff9fd615",
+    "1cc628533d0004b2b20e7f4baad0b8bb5e0673db159bbccf92491aef61fc9620",
+    "880e0bbf82a8cf818ed46ba03cf0fc6c898e36fca36cc7fdb1d2db7503634430"},
+    {NID_X9_62_prime256v1, NID_sha256,
+    "f8248ad47d97c18c984f1f5c10950dc1404713c56b6ea397e01e6dd925e903b4fadfe2c9"
+    "e877169e71ce3c7fe5ce70ee4255d9cdc26f6943bf48687874de64f6cf30a012512e787b"
+    "88059bbf561162bdcc23a3742c835ac144cc14167b1bd6727e940540a9c99f3cbb41fb1d"
+    "cb00d76dda04995847c657f4c19d303eb09eb48a",
+    "b20d705d9bd7c2b8dc60393a5357f632990e599a0975573ac67fd89b49187906",
+    "0451f99d2d52d4a6e734484a018b7ca2f895c2929b6754a3a03224d07ae61166ce4737da"
+    "963c6ef7247fb88d19f9b0c667cac7fe12837fdab88c66f10d3c14cad1",
+    "72b656f6b35b9ccbc712c9f1f3b1a14cbbebaec41c4bca8da18f492a062d6f6f",
+    "9886ae46c1415c3bc959e82b760ad760aab66885a84e620aa339fdf102465c42",
+    "2bf3a80bc04faa35ebecc0f4864ac02d349f6f126e0f988501b8d3075409a26c"},
+    {NID_X9_62_prime256v1, NID_sha256,
+    "3b6ee2425940b3d240d35b97b6dcd61ed3423d8e71a0ada35d47b322d17b35ea0472f35e"
+    "dd1d252f87b8b65ef4b716669fc9ac28b00d34a9d66ad118c9d94e7f46d0b4f6c2b2d339"
+    "fd6bcd351241a387cc82609057048c12c4ec3d85c661975c45b300cb96930d89370a327c"
+    "98b67defaa89497aa8ef994c77f1130f752f94a4",
+    "d4234bebfbc821050341a37e1240efe5e33763cbbb2ef76a1c79e24724e5a5e7",
+    "048fb287f0202ad57ae841aea35f29b2e1d53e196d0ddd9aec24813d64c0922fb71f6daf"
+    "f1aa2dd2d6d3741623eecb5e7b612997a1039aab2e
```

Comment:
"# endif"

Reply:
Thanks! Missed `openssl-format-source` on the header file -- fixup in 60bcac5e3ffa8f625373b64b7523d2dcc022d93d